libs/cardano-ledger-pretty/src/Cardano/Ledger/Pretty.hs

```diff
@@ -682,12 +682,19 @@ ppInstantaneousRewards (InstantaneousRewards res treas dR dT) =
     , ("deltaTreasury", ppDeltaCoin dT)
     ]
 
-ppShelleyPPUPState :: PrettyA (PParamsUpdate era) => ShelleyPPUPState era -> PDoc
-ppShelleyPPUPState (ShelleyPPUPState p fp) =
+ppShelleyGovState ::
+  ( PrettyA (PParamsUpdate era)
+  , PrettyA (PParams era)
+  ) =>
+  ShelleyGovState era ->
+  PDoc
+ppShelleyGovState (ShelleyGovState p fp pp prev) =
   ppRecord
-    "Proposed ShelleyPPUPState"
+    "Proposed ShelleyGovState"
     [ ("proposals", ppProposedPPUpdates p)
     , ("futureProposals", ppProposedPPUpdates fp)
+    , ("prevPParams", prettyA prev)
+    , ("currentPParams", prettyA pp)
```

Comment:
For consistency sake with `prev`, otherwise it feels unfair to the previous pparams :grin: 
```suggestion
    , ("curPParams", prettyA pp)
```